ckanext/geocat/utils/vocabulary_utils.py

```diff
@@ -36,3 +41,9 @@ def get_terms_of_use():
     file = os.path.join(__location__, 'geocat-terms-of-use.xml')
     g.parse(file, format='xml')
     return g
+
+
+def validate_terms_of_use(terms_of_use):
+    if terms_of_use in [TERMS_OF_USE_BY, TERMS_OF_USE_ASK, TERMS_OF_USE_BY_ASK, TERMS_OF_USE_OPEN]:
+        return True
+    return False
```

Comment:
This is fine, but imo it would be neater to
- define the array of possible terms of use as a constant as well (at the top of the file after the other ones are defined)
- simplify the body of the function to `return terms_of_use in TERMS_OF_USE`